makefiles/Makefile.clang

```diff
@@ -8,6 +8,7 @@ else
 	CXXFLAGS  := -O3 -fPIC -DUNIX -Wall -g -std=c++11
 endif
 OMPFLAGS=-fopenmp
+OMPOFFLOADFLAGS=-fopenmp -fopenmp-targets=nvptx64-nvidia-cuda -Xopenmp-target -march=sm_75
```

Comment:
Please have these respect NV_ARCH, like in case of CUDA

Reply:
Hi @reguly, I have push the changes now. Please check the updated PR.